### PR TITLE
added tutorial notebook

### PR DESCRIPTION
**Pull request by bwh33l3r**
added a tutorial notebook that goes through the process of ingesting an ISIS control network file and image file list into a NetworkCandidateGraph. Also removed ipython extension for gitignore.

### CODE REVIEW

File: docs/users/tutorials/from_cnet.ipynb [hunk 1]
```diff
@@ -0,0 +1,1408 @@
...
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from plio.io.io_controlnetwork import from_isis\n",
+    "\n",
+    "cnet_path = '/work/projects/kaguya/ladoramkershner/Control/imagenet/Ground/grounded_rr_jig1_ERR_PROP.net'\n",
```
Comment (jessemapel) [inline docs/users/tutorials/from_cnet.ipynb:54]: This is a local path on our systems, for tutorials we need to use simpler paths. I would move this control network file into the same directory as this notebook and give it a simple name such as "Kaguya_TC_with_ground.net".

File: docs/users/tutorials/from_cnet.ipynb [hunk 2]
```diff
@@ -0,0 +1,1408 @@
...
+      "             'queue': 'shortall',\n",
+      "             'tmp_scratch_dir': '/scratch'},\n",
+      " 'database': {'host': 'autocnet.wr.usgs.gov',\n",
+      "              'name': 'bhw_isis_ingest_tutorial',\n",
+      "              'password': 'abcde',\n",
+      "              'pgbouncer_port': 30001,\n",
+      "              'port': 5432,\n",
+      "              'timeout': 500,\n",
+      "              'type': 'postgresql',\n",
+      "              'username': 'jay'},\n",
```
Comment (jessemapel) [inline docs/users/tutorials/from_cnet.ipynb:823]: We do not want to have information about login credentials in the tutorial. I would not print out the contents of your config file in this notebook.

File: docs/users/tutorials/from_cnet.ipynb [hunk 3]
```diff
@@ -0,0 +1,1408 @@
...
+    "cnet_path = \"/work/projects/kaguya/ladoramkershner/Control/imagenet/Ground/grounded_rr_jig1_ERR_PROP.net\"\n",
+    "\n",
+    "lis_path = \"/scratch/jmapel/autocnet_tut/cubes.lis\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "-Next cell is the real meat, ingesting the images. \n",
```
Comment (jessemapel) [inline docs/users/tutorials/from_cnet.ipynb:885]: This list is not rendering correctly, you need to add a space after the dash:

```
-this doesn't work
- this don't
```

File: docs/users/tutorials/from_cnet.ipynb [hunk 4]
```diff
@@ -0,0 +1,1408 @@
...
+   "source": [
+    "from autocnet.io.db.model import Images, Points\n",
+    "with ncg.session_scope() as session:\n",
+    "    res = session.query(Points).all()\n",
+    "    print(len(res))\n",
+    "    for i in range(0,11):\n",
+    "        print(f'point {res[i].id} geom: ', res[i].geom)"
+   ]
+  }
+ ],
```
Comment (jessemapel) [inline docs/users/tutorials/from_cnet.ipynb:1386]: These last two cells haven't been run. We should have the full notebook run with all of the cells' outputs.

**Review by jessemapel — COMMENTED** [review]
Looks good. You should remove local paths on our system and do all paths relative to the notebook. You should also remove the database login credentials so that they are not public.

Finally, we should have all of the cells run with their output. I am concerned about the size of this file with such a large control network. We should find a smaller control network with more reasonable outputs.

**Comment by bwh33l3r** [comment]
@ladoramkershner the data to run the notebook is located at: /home/bhwheeler/work_users/autocnet/docs/users/tutorials/isis_ingestion

All of the point geom come in at 0,90.  Do you know why this is happening?

**Comment by ladoramkershner** [comment]
@bwh33l3r In the notebook's networks (AS15_landingsite_apollolro.net and AS15_LandingSide_Metric-NAC-LO5H-FIN.net) there is no ground (x,y,z surface coordiates) information in the point groups. I used cnetbin2pvl to turn the networks into a readable format and here is the information the networks have on a point.

```
Object = ControlPoint
    PointType   = Free
    PointId     = M102128467LE_bndry_11
    ChooserName = cnetmerge
    DateTime    = 2014-03-17T15:07:59
End_Object
```

Since there is no information for ground location, plio defaults to (x, y, z) = (0, 0, 0) which results in (lon, lat) = (0, 90)

How did you make and/or where did you get these networks?

**Comment by jessemapel** [comment]
@ladoramkershner This network is from one of the jigsaw tests. It was originally made ~5 years ago by tammy and ken

**Comment by ladoramkershner** [comment]
Do you remember the original location of the network?

**Comment by jessemapel** [comment]
@ladoramkershner `/usgs/cpkgs/isis3/isis_testData/isis/src/control/apps/jigsaw/tsts/multipleSensors/input` that directory also contains some diff files for testing that you can ignore.

**Review by ladoramkershner — APPROVED** [review]

**Comment by ladoramkershner** [comment]
@jessemapel are we comfortable merging this and checking the nightly doc builds?

**Review by jessemapel — APPROVED** [review]

**Comment by jessemapel** [comment]
@ladoramkershner Yes, for the sake of expediency, let's let the nightly doc build go.

**Comment by jessemapel** [comment]
Travis timed out solving for the 3.6 environment, but this is good to go.

**Pull request merged**